### ISSUE TITLE
Template selector stopped working on v1.2.0

### ISSUE THREAD

**Issue by vladcampos** [numeroflip/obsidian-auto-template-trigger#16]
Hello, I'm not sure if I'm doing something wrong or if I've found a bug. After upgrading to v1.2.0, the template selector stopped popping up when I create a new note. I tested it on another vault where I still have v1.1.3 installed, and it is working perfectly fine.

I'm running Obsidian 1.5.12 (Installer 1.4.13)

**Comment by vladcampos** [comment]
@numeroflip I'm glad it was helpful. I love your plugin. I use it every single day and have already talked about it in some of my videos 😉

**Comment by numeroflip** [comment]
@vladcampos Ohh, thank you for your kind words. :) 
Off-topic: Have you seen the new feature of folder templates? The plugin now has a settings tab for that, and I'm wondering if the community would find that useful. 

Also, I'd love to see a video where this plugin is used, so feel free to post one if you don't mind. 

I believe the original issue is resolved now, right? I'm closing it. If you notice anything else, feel free to open a new one.

**Comment by vladcampos** [comment]
I am currently working on a video to demonstrate the folder templates, and I believe that I may have found another bug.

At least here, it is only automatically creating the selected template in folders and subfolder. If I set it at deeper levels, sub-subfolder for example, it doesn't add the template and shows the pop-up window to select one from the list.

PARENT (here it woks)
 |_ ONE (here it woks)
      |__ TWO (from here on, it doesn't work anymore)

**Comment by numeroflip** [comment]
@vladcampos :  Hmm, that's right, I haven't tested deeper nesting, thanks for catching that :) . I found the bug causing a similar issue, and `1.2.2` is already out in which it is fixed. 

Although... I could not reproduce the exact issue you mentioned. For me the pop-up wasn't shown in a sub-subfolder, but the upper level template was applied automatically. Can you maybe provide a step by step description on how to reproduce the issue? 

What I've tried: 
1. Created a folder structure like: 
```
ROOT
|_ Folder
     |_ Subfolder
           |_SubSubfolder 
``` 

2. Created templates: `Root.md`, `Folder.md`, `Subfolder.md` `SubSubfolder.md` in a Templates folder
3. Assigned the templates in the settings: 
```
/                              -> Template: Root
Folder                         -> Template: Folder
Folder/Subfolder               -> Template: Subfolder 
Folder/Subfolder/SubSubfolder  -> Template: SubSubfolder
```
Created files in each of the folders:
```
/test.md                               -> Template: Root : applied ✅
Folder/test.md                         -> Template: Folder: applied ✅
Folder/Subfolder/test.md               -> Template: Subfolder:  applied ✅
Folder/Subfolder/SubSubfolder/test.md  -> Template: SubSubfolder: NOT applied ⛔. No poups. Instead the Subfolder template was automatically applied

```
                   

I really need to find a way to set up testing, so bugs like this are found before releasing a version

**Comment by vladcampos** [comment]
I know why I was getting the pop-up. It's because I was doing it on my testing account, setting only one folder or subfolder at a time. Since there was nothing above to choose from, it was showing me the pop-up. Before updating, I added rules to the above folders, and it behaved just like you described. In other words, always getting the template from the folder above.

After that, I updated it to the newest version and everything is working fine 😉 

This folder feature is a game-changer. Thank you for implementing it. Maybe I'll be able to publish the video today. It will depend on my schedule. But if not today, in will happen the next couple of days.
